Alpine's eudev does not add the TAG+=seat to render / card DRI devices;
fix found here: https://wiki.gentoo.org/wiki/User:Tlhonmey

also, it's dropped in https://gitlab.freedesktop.org/plymouth/plymouth/-/commit/dbec4487 anyway,
and will be included in the next plymouth release

diff --git a/src/libply-splash-core/ply-device-manager.c b/src/libply-splash-core/ply-device-manager.c
index aed7bac..859bce4 100644
--- a/src/libply-splash-core/ply-device-manager.c
+++ b/src/libply-splash-core/ply-device-manager.c
@@ -353,16 +353,12 @@ create_devices_for_subsystem (ply_device_manager_t *manager,
                         /* We only care about devices assigned to a (any) devices. Floating
                          * devices should be ignored.
                          */
-                        if (udev_device_has_tag (device, "seat")) {
                                 const char *node;
                                 node = udev_device_get_devnode (device);
                                 if (node != NULL) {
                                         ply_trace ("found node %s", node);
                                         found_device = create_devices_for_udev_device (manager, device);
                                 }
-                        } else {
-                                ply_trace ("device doesn't have a devices tag");
-                        }
                 } else {
                         ply_trace ("it's not initialized");
                 }
